config/routes.rb

```diff
@@ -3,6 +3,6 @@
 BrowseEverything::Engine.routes.draw do
   get 'connect', to: 'browse_everything#auth', as: 'connector_response'
   match 'resolve', to: 'browse_everything#resolve', as: 'resolver', via: %i[get post]
-  match ':provider(/*path)', to: 'browse_everything#show', as: 'contents', via: %i[get post]
+  match ':provider(/*path)', to: 'browse_everything#show', as: 'contents', via: %i[get post], format: false
```

Comment:
A comment here would be useful. It's not immediately evident why no format options should be allowed.